### PR TITLE
Update apt-by-hash docs

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

File: docs/feature_overview.rst [hunk 1]
```diff
@@ -153,11 +153,13 @@ AptByHash is a feature that mitigates commonplace 'Hash Sum Mismatch' errors dur
 It adds the checksum of the package metadata to the the packages' names.
 These files are then stored within a 'by-hash' directory within each release architecture in the specified debian repository.
 The client will then use the filename to identify the expected checksum and download a file whose name matches the checksum.
+The pulp_deb plugin will cache and continue to serve previous publications' published artifacts
+including the by-hash files for up to 3 days so that users can still fetch these files after a new
+publication is created.
```
Comment (mdellweg) [inline docs/feature_overview.rst:158]: That is if the publication is not removed by anything, right?

Comment (daviddavis) [reply docs/feature_overview.rst:158]: Yes, I'll add a note about that.

**Review by quba42 — APPROVED** [review]
I think all of this is correct for the latest version of pulp_deb.

**Review by hstct — APPROVED** [review]
LGTM we can merge this together with #1061

**Comment by quba42** [comment]
I think we can merge this now, since it is already true for the current level of apt-by-hash support (without source files).

**Pull request merged**